engines/query-sparql-solid/package.json

```diff
@@ -44,139 +44,21 @@
-    "@comunica/actor-rdf-metadata-extract-sparql-service": "^2.5.1",
-    "@comunica/actor-rdf-metadata-primary-topic": "^2.5.1",
-    "@comunica/actor-rdf-parse-html": "^2.5.1",
-    "@comunica/actor-rdf-parse-html-microdata": "^2.5.1",
-    "@comunica/actor-rdf-parse-html-rdfa": "^2.5.1",
-    "@comunica/actor-rdf-parse-html-script": "^2.5.1",
-    "@comunica/actor-rdf-parse-jsonld": "^2.5.1",
-    "@comunica/actor-rdf-parse-n3": "^2.5.1",
-    "@comunica/actor-rdf-parse-rdfxml": "^2.5.1",
-    "@comunica/actor-rdf-parse-xml-rdfa": "^2.5.1",
-    "@comunica/actor-rdf-resolve-hypermedia-links-next": "^2.5.1",
-    "@comunica/actor-rdf-resolve-hypermedia-links-queue-fifo": "^2.5.1",
-    "@comunica/actor-rdf-resolve-hypermedia-none": "^2.5.1",
-    "@comunica/actor-rdf-resolve-hypermedia-qpf": "^2.5.1",
-    "@comunica/actor-rdf-resolve-hypermedia-sparql": "^2.5.1",
-    "@comunica/actor-rdf-resolve-quad-pattern-federated": "^2.5.1",
-    "@comunica/actor-rdf-resolve-quad-pattern-hypermedia": "^2.5.1",
-    "@comunica/actor-rdf-resolve-quad-pattern-rdfjs-source": "^2.5.1",
-    "@comunica/actor-rdf-resolve-quad-pattern-string-source": "^2.5.1",
-    "@comunica/actor-rdf-serialize-jsonld": "^2.5.1",
-    "@comunica/actor-rdf-serialize-n3": "^2.5.1",
-    "@comunica/actor-rdf-update-hypermedia-patch-sparql-update": "^2.5.1",
-    "@comunica/actor-rdf-update-hypermedia-put-ldp": "^2.5.1",
-    "@comunica/actor-rdf-update-hypermedia-sparql": "^2.5.1",
-    "@comunica/actor-rdf-update-quads-hypermedia": "^2.5.1",
-    "@comunica/actor-rdf-update-quads-rdfjs-store": "^2.5.1",
-    "@comunica/bus-http-invalidate": "^2.5.1",
-    "@comunica/config-query-sparql": "^2.2.0",
-    "@comunica/context-entries": "^2.5.1",
-    "@comunica/core": "^2.5.1",
-    "@comunica/logger-void": "^2.5.1",
-    "@comunica/mediator-all": "^2.5.1",
-    "@comunica/mediator-combine-pipeline": "^2.5.1",
-    "@comunica/mediator-combine-union": "^2.5.1",
-    "@comunica/mediator-join-coefficients-fixed": "^2.5.1",
-    "@comunica/mediator-number": "^2.5.1",
-    "@comunica/mediator-race": "^2.5.1",
-    "@comunica/runner": "^2.5.1",
-    "@comunica/runner-cli": "^2.5.1",
-    "@rubensworks/solid-client-authn-isomorphic": "^2.0.0",
-    "solid-node-interactive-auth": "^1.0.2"
+    "@comunica/actor-init-query": "^2.9.0",
+    "@comunica/config-query-sparql-solid": "^2.4.0",
+    "@comunica/context-entries": "^2.8.2",
+    "@comunica/core": "^2.8.2",
+    "@comunica/query-sparql": "^2.9.0",
+    "@comunica/runner-cli": "^2.8.2",
+    "@rubensworks/solid-client-authn-isomorphic": "^2.0.1",
+    "solid-node-interactive-auth": "^1.1.1"
   },
   "scripts": {
     "build": "npm run build:ts",
     "build:ts": "node \"../../node_modules/typescript/bin/tsc\"",
     "prepare": "comunica-compile-config config/config-default.json > engine-default.js",
-    "browser": "npm run prepare && \\\"../../node_modules/webpack/bin/webpack.js\\\" --config webpack.config.js --mode production"
```

Comment:
Any reason why you changed this? I think this was like this because of Windows.

Reply:
Yes, it throws an error when there is the extra `\` before quotes:
```
yarn run v1.22.19
$ npm run prepare && \"../../node_modules/webpack/bin/webpack.js\" --config webpack.config.js --mode production

> @comunica/query-sparql-solid@2.4.0 prepare
> comunica-compile-config config/config-default.json > engine-default.js

/bin/sh: line 1: "../../node_modules/webpack/bin/webpack.js": No such file or directory
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
But I have reverted it to how it was for now in case it is needed on Windows, I cannot test on Windows myself.

Reply:
Let's just do the same as base comunica: https://github.com/comunica/comunica/blob/master/engines/query-sparql/package.json#L186

Reply:
Okay, now it is the same as base Comunica. I can look into the browser tests later.